client/tsconfig.json

```diff
@@ -13,7 +13,7 @@
     "sourceMap": true,                        /* Generates corresponding '.map' file. */
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
     "outDir": "./lib/",                       /* Redirect output structure to the directory. */
-    "rootDir": "./functions/src",             /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
+    "rootDir": "../",                         /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
```

Comment:
I think if you made `rootDir` `"../client/"` it would fix the `src` sub-folder in the `lib` folder.

Reply:
I guess you meant "./src" (this file is already in client dir). I tried that. But it needs to be the main folder of the repository, as the client is using files from `functions` directory too. So the root needs to include that too.